app/about_us/OurCrew.js

```diff
@@ -0,0 +1,61 @@
+import styles from "./page.module.css";
+
+const OurCrew = () => {
+  return (
+    <>
+      <p>
+        Our crew is the heart and soul of Galactica. We are a diverse team of
+        seasoned space explorers, engineers, and visionaries who are united by a
+        common goal: to make space travel accessible and exciting for all.
+      </p>
+      <div className={styles.ourCrew}>
+        <li>
```

Comment:
You could replace these `<li>` by a component that takes 2 props: `image` and `description`, e.g. `<OurCrewMember image="path/to/image.png" description="lorem ipsum" />`

Reply:
Also remember the good practices of HTML: a `<li>` can only be the child of a list element.